automation/test.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018 Red Hat, Inc.
+#
+
+# CI considerations: $TARGET is used by the jenkins build, to distinguish what to test
+# Currently considered $TARGET values:
+#     kubernetes-release: Runs all functional tests on a release kubernetes setup
+#     openshift-release: Runs all functional tests on a release openshift setup
+
+set -ex
+
+export WORKSPACE="${WORKSPACE:-$PWD}"
+readonly ARTIFACTS_PATH="$WORKSPACE/exported-artifacts"
+
+if [[ $TARGET =~ openshift-.* ]]; then
+  export KUBEVIRT_PROVIDER="os-3.11.0-multus"
+else
+  export KUBEVIRT_PROVIDER="k8s-multus-1.12.2"
+fi
+
+kubectl() { cluster/kubectl.sh "$@"; }
+
+export NAMESPACE="${NAMESPACE:-kube-system}"
+
+# Make sure that the VM is properly shut down on exit
+trap '{ make cluster-down; }' EXIT SIGINT SIGTERM SIGSTOP
+
+make cluster-down
+make cluster-up
+
+echo "Nodes are ready:"
+kubectl get nodes
+
+# OpenShift is running important containers under default namespace
+namespaces=(kube-system default)
+if [[ $NAMESPACE != "kube-system" ]]; then
+  namespaces+=($NAMESPACE)
+fi
+
+# Run cluster-sync to deploy bridge-marker on the nodes
+make cluster-sync
+
+timeout=300
+sample=30
+
+for i in ${namespaces[@]}; do
+
+  # Make sure all containers are ready
+  current_time=0
+  while [ -n "$(kubectl get pods -n $i -o'custom-columns=status:status.containerStatuses[*].ready' --no-headers | grep false)" ]; do
+    echo "Waiting for KubeVirt containers to become ready ..."
```

Comment:
we don't deploy any kubevirt compononents, please drop this blocking code.

Reply:
removed